README.md

```diff
@@ -21,8 +21,10 @@
 
 # 📖 [Enumerable] <a name="about-project"></a>
 
+
 *# OOP_school_library* is a Ruby project that involves creating a Ruby App and Building a library for a school representing students and teachers. Later you can add more details like borrowing of books, parents permission to students younger than 18 years etc.
 
```

Comment:
### Optional

- [ ] Let's polish our readme file and make it appealing to potential recruiters
  - Improve titles to better show what our project is about 🌟 .    
![image](https://github.com/AbuTalha3/school-library-oop/assets/98453979/2f72d050-98c1-4f51-ab1d-55c4a357db9a)

  - lets remove duplicated headers

![image](https://github.com/AbuTalha3/school-library-oop/assets/98453979/c05016a1-28f9-4e31-8b21-125fe14d3ccf)